### PR TITLE
outputs changes for cloud9 integration

### PR DESCRIPTION
**Pull request by snjkumar23**
## What does this PR do and why?

Changes in outputs section for cloud9 integration

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [ ] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by rezabekf — CHANGES_REQUESTED** [review]
See the inline feedback

File: content/basics/templates/outputs/index.md
Comment (rezabekf) [inline content/basics/templates/outputs/index.md:55]:
```suggestion
1. Create the stack with a new template.
```

File: content/basics/templates/outputs/index.md
Comment (rezabekf) [inline content/basics/templates/outputs/index.md:70]: Please fix to cfn-create.

Please fix the naming to `cfn-workshop-<lab-name>`

File: content/basics/templates/outputs/index.md
Comment (rezabekf) [inline content/basics/templates/outputs/index.md:85]: Could you please update this section to create new stack. This was previously wrongly assuming to update stack where we want to create new one.

File: content/basics/templates/outputs/index.md
Comment (rezabekf) [inline content/basics/templates/outputs/index.md:142]: please fix stack name `cfn-workshop-<lab-name>`

File: content/basics/templates/outputs/index.md
Comment (rezabekf) [inline content/basics/templates/outputs/index.md:149]: please fix stack name `cfn-workshop-<lab-name>`

**Review by rezabekf — CHANGES_REQUESTED** [review]
Please see the comment inline. The lab should be independent from resources lab. We have previously wrongly assumed, that resources lab is completed.

File: content/basics/templates/outputs/index.md
Comment (snjkumar23) [reply content/basics/templates/outputs/index.md]: incorporated

File: content/basics/templates/outputs/index.md
Comment (rezabekf) [reply content/basics/templates/outputs/index.md]: I will do this.

File: content/basics/templates/outputs/index.md
Comment (sdangol) [inline content/basics/templates/outputs/index.md:65]:
```suggestion
:::code{language=shell showLineNumbers=false showCopyAction=true}
```

File: content/basics/templates/outputs/index.md
Comment (sdangol) [inline content/basics/templates/outputs/index.md:136]:
```suggestion
  :::code{language=shell showLineNumbers=false showCopyAction=true}
```

File: content/basics/templates/outputs/index.md
Comment (rezabekf) [inline content/basics/templates/outputs/index.md:151]:
```suggestion
1. In **Specify template**, choose **Upload a template file**.
```

File: content/basics/templates/outputs/index.md
Comment (rezabekf) [inline content/basics/templates/outputs/index.md:78]:
```suggestion
1. In **Specify template**, choose **Upload a template file**.
```

Comment (rezabekf) [inline content/basics/templates/outputs/index.md:78]:
```suggestion
1. In **Specify template**, choose **Upload a template file**.
```

File: content/basics/templates/outputs/index.md
Comment (rezabekf) [inline content/basics/templates/outputs/index.md:155]:
```suggestion
1. On the **Review <stack_name>** page, scroll down to the bottom and click on **Submit**.
```

**Review by rezabekf — CHANGES_REQUESTED** [review]
Small changes required.

**Review by rezabekf — CHANGES_REQUESTED** [review]
Please resolve all the changes requested. There is quite a few required to do second review.

**Comment by snjkumar23** [comment]
> Please resolve all the changes requested. There is quite a few required to do second review.

I resolved all changes already as per your review comments, are'nt they visible in latest commit

**Comment by rezabekf** [comment]
Sorry @snjkumar23 , GitHub was down yesterday and was behaving weirdly, so I couldn't see the request has been implemented...looking now...

**Review by rezabekf — APPROVED** [review]
LGMT

**Pull request merged**
